src/app_neo4j_queries.py

```diff
@@ -399,6 +459,76 @@ def get_next_revisions(neo4j_driver, uuid, property_key = None):
 
     return results
 
+
+def is_next_revision_latest(neo4j_driver, uuid, property_key = None):
```

Comment:
We don't need this `property_key` as an optional argument in the method signature at all. Modify the query to only return a list of uuids, then move the non-empty list check here, and return a boolean as the final result. When function is named as `is_xxx` a boolean return is expected as the norm.